install/install.py

```diff
@@ -113,11 +113,21 @@ def finish():
 
 
 def setup_virtualenv():
-    Colorizer.info("[*] Seting up virtual environment named owtf...")
+    Colorizer.info("[*] Setting up virtual environment named owtf...")
+    # If /usr/local/bin/virtualenvwrapper.sh doesn't exist, create a symlink from /usr/bin/
+    if not os.path.isfile('/usr/local/bin/virtualenvwrapper.sh'):
+        run_command('sudo ln -s /usr/bin/virtualenvwrapper.sh /usr/local/bin/virtualenvwrapper.sh >/dev/null 2>&1;')
     # sources files and commands
     source = 'source /usr/local/bin/virtualenvwrapper.sh'
-    setup_env = 'cd $WORKON_HOME; virtualenv -q --always-copy -p %s owtf >/dev/null 2>&1;'\
-        ' source owtf/bin/activate' % sys.executable
+
+    # Workaround for systems where the default Python Interpreter is 3.0+
+    if not os.path.isfile('/usr/bin/virtualenv2'):
+        setup_env = 'cd $WORKON_HOME; virtualenv -q --always-copy -p %s owtf >/dev/null 2>&1;'\
+            ' source owtf/bin/activate' % sys.executable
+    else:
+        setup_env = 'cd $WORKON_HOME; virtualenv2 -q --always-copy -p %s owtf >/dev/null 2>&1;'\
```

Comment:
It seems very redundant to copy/paste the line twice when the only difference is the additional `2` after `virtualenv`.

Could you instead use `%s` or `.format()`?